src/hiero_sdk_python/query/transaction_record_query.py

```
@@ -51,29 +95,55 @@ def _make_request(self):
+        if not self.transaction_id:
+            raise ValueError("Transaction ID must be set before making the request.")
+
+        query_header = self._make_request_header()
+        transaction_get_record = transaction_get_record_pb2.TransactionGetRecordQuery()
+        transaction_get_record.header.CopyFrom(query_header)
+        transaction_get_record.transactionID.CopyFrom(self.transaction_id._to_proto())
+        transaction_get_record.includeDuplicates = self.include_duplicates
+
+        query = query_pb2.Query()
+        if not hasattr(query, "transactionGetRecord"):
+            raise AttributeError("Query object has no attribute 'transactionGetRecord'")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)